user_tools/src/spark_rapids_tools/configuration/common.py

```diff
@@ -23,33 +23,28 @@
 
 
 class RuntimeDependencyType(BaseModel):
-    """Defines the type of dependency. It can be one of the following:
-       - Archived file (.tgz)
-       - Simple JAR file (*.jar)
-       - Classpath directory (not yet supported)
-
-    Note: The 'classpath' type is reserved for future use, allowing users to point to a directory
-    in the classpath without needing to download or copy any binaries."""
+    """Defines the type of runtime dependency required by the tools' java cmd."""
 
     dep_type: DependencyType = Field(
-        description='The type of the dependency',
+        description='The type of the dependency.',
         validation_alias=AliasChoices('dep_type', 'depType'))
     relative_path: str = Field(
         default=None,
-        description='The relative path of the dependency in the classpath. This is relevant for tar files',
+        description='Specifies the relative path from within the archive file which will be added to the java cmd. '
+                    'Requires field dep_type to be set to (archive).',
         validation_alias=AliasChoices('relative_path', 'relativePath'),
-        examples=['jars/*'])
+        examples=['*'])
```

Comment:
Q: I am not sure if this helps me understand what `relative_path` should be?

Reply:
Fixed putting back `jars/*`

There is description on `dependency_type`.

> "Specifies the dependency type to determine how the item is processed. For example, jar files are appended to the java classpath while archive files such as spark are extracted first before adding subdirectory _/jars/* to the classpath"

Do you have a suggestion on a better way to document that? The problem is that adding too much details make the file hard to read.

Reply:
thanks @amahussein! this looks better to me.